src/modules/charge-versions/routes/charge-version-workflows.js

```diff
@@ -22,7 +22,10 @@ module.exports = {
         scope: [chargeVersionWorkflowEditor, chargeVersionWorkflowApprover]
       },
       validate: {
-        headers
+        headers,
```

Comment:
If it's OK with you, I think we should probably rename headers to something like `headersValidation` or something like that. 

Not important, just a readability thing.

Reply:
We've called it `headers` across many routes files. So I'm inclined to leave this as is